test/commonTests.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-env mocha */
+/* global expect */
+
+import nock from 'nock'
+
+export function notifiesWithUsageMessageForDashH() {
+  const {lgJWT, lgUser} = this
+  return this.invoke(['-h'], this.notify, {lgJWT, lgUser})
+    .then(() => {
+      expect(this.notifications[0]).to.match(/Usage:/)
+    })
+}
+
+export function notifiesWithUsageHintForInvalidArgs(invalidArgv) {
```

Comment:
What about going one step further and doing this?

```javascript
export function itNotifiesWithUsageHintForInvalidArgs(invalidArgv) {
  it(`notifies with a usage hint when invoked with invalid args`, function () {
    const {lgJWT, lgPlayer} = this
    // ...
  })
}
```

That would make for less duplication when calling these common functions and we wouldn't have to do the `bind(this)` stuff when calling them. We would have to accept a "thunk" for args where we want to be able to reference `this` so this:

```javascript
it('notifies with an error message when invoked by a non-player', function () {
      return notifiesWithErrorIfNotAPlayer(this.argv).bind(this)()
})
```

would become this:

```javascript
  itNotifiesWithErrorIfNotAPlayer(function() { this.argv })
```

So not sure if that's better. Anyway, just an idea.